index.html

```diff
@@ -22,12 +23,18 @@ <h2>Awesome book Library</h2>
            
         </tbody>
     </table>
-    <form  id="form-Book">
-        <input type="text" name="Author" id="authorname" placeholder="Author"><br><br>
-        <input type="text" name="Title" id="titlename" placeholder="Title"><br><br>
+    <form  id="form-Book" class="form-book">
+        <label for="author">Author</label>
+        <input type="text" name="Author" id="authorname" placeholder="Author">
+        <br><br>
+        <label for="title">Title</label>
+        <input type="text" name="Title" id="titlename" placeholder="Title">
+        <br><br>
        
         <input type="submit" value="Add" id="addbutton" >
     </form>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you add the `required` attribute in all your `<input>` tags for validation so the user cannot add an empty form.